synapse/lib/storm.py

```diff
@@ -230,10 +230,25 @@
                     'perm': {'type': 'array', 'items': {'type': 'string'}},
                     'desc': {'type': 'string'},
                     'gate': {'type': 'string'},
+                    'workflowconfig': {'type': 'boolean'},
                 },
                 'required': ['perm', 'desc', 'gate'],
             },
         },
+        'configvars': {
+            'type': 'array',
+            'items': {
+                'type': 'object',
+                'properties': {
+                    'name': {'type': 'string'},
+                    'varname': {'type': 'string'},
+                    'desc': {'type': 'string'},
+                    'default': {'type': 'string'},
+                    'workflowconfig': {'type': 'boolean'},
```

Comment:
We may need a way to permit different scopes in an extensible way. global / self ( and maybe a future "role"? )